src/task.c

```diff
@@ -978,15 +1041,17 @@ void jl_init_root_task(void *stack_lo, void *stack_hi)
     arraylist_new(&ptls->current_task->locks, 0);
 #endif
 
+#ifdef COPY_STACKS
```

Comment:
Probably simpler to make sure that `always_copy_stack == 0` if we are running on WASM.
Also would fix my comment above, c.f. 48634f9f8669e1dc1be0a1589cd5be880c04055a

Reply:
We want to avoid the syntactic setjmp here, because it isn't available on wasm.